Updates/0361_call_of_water.sql

```diff
@@ -0,0 +1,293 @@
+-- Remove script from Quest 63
+DELETE FROM dbscripts_on_quest_end WHERE id = '63';
+UPDATE `quest_template` SET `CompleteScript`='0' WHERE `entry`='63';
+
+-- QuestID 100 'Call of Water' Start Script
+DELETE FROM dbscripts_on_quest_start WHERE id = '100';
+INSERT INTO dbscripts_on_quest_start(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+(100,0,0,45,100,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Start Relay Script');
+
+UPDATE `quest_template` SET `StartScript`='100' WHERE `entry`='100';
+
+-- QuestID 1103 Call of Water End Script
+DELETE FROM dbscripts_on_quest_end WHERE id = '1103';
+INSERT INTO dbscripts_on_quest_end(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+(1103,0,0,28,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Tiev Mordune - Remove Standstate'),
+(1103,0,0,21,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Tiev Mordune - Set Active Object'),
+(1103,1000,0,0,0,0,0,0,0,0,3069,0,0,0,0,0,0,0,0,0,0,'Call of Water - Tiev Mordune - Say Text'),
+(1103,4000,0,20,2,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Tiev Mordune - Change Movement');
+
+-- QuestID 96 Start Script
+DELETE FROM dbscripts_on_quest_start WHERE id = '96';
+INSERT INTO dbscripts_on_quest_start(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+(96,20000,0,45,96,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Start Relay Script');
+
+UPDATE `quest_template` SET `StartScript`='96' WHERE `entry`='96';
+
+-- Relay Scripts
+DELETE FROM dbscripts_on_relay WHERE id IN (96, 100, 5897);
+INSERT INTO dbscripts_on_relay(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+-- This relay Script starts when quest 96 is accepted or 5 minutes after the Corrupted Water Spirit got spawned
+(96,0,3,29,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Remove NPC Flags'),
+(96,0,2,20,2,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Start Movement'),
+(96,0,1,35,5,0,0,6748,50,512,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Send AI Event'),
+-- Spawn script
+(100,0,1,35,5,50,0,5897,50,512,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Corrupt Water Spirit of Water - Send AI Event'),
```

Comment:
DB Error: 
2022-07-07 17:34:55  DB-SCRIPTS: Process table `dbscripts_on_relay` id 100, command 35 has buddy 5897 not found in range 50 of searcher Creature (Entry: 7007 Guid: 17627) (data-flags 512), skipping.
When they are already dead.

Reply:
Any reason quests 96 and 100 have to start relay scripts instead of using dbscripts on queststart?

Reply:
No real reason just less code.
relay 96 gets called on quest end, and from creature_ai_script (5 minutes after spawned)
realy 100 gets called from quest 100 and on creature_movement